scroll-list.lua

```diff
@@ -15,6 +15,9 @@ local scroll_list = {
     empty_text = "no entries"
 }
 
+local opts = require("mp.options")
+opts.read_options(scroll_list, "scroll_list", function(list) update_opts(list) end)
+
```

Comment:
I don't really want these options to be exposed to people. Scroll-list is an API, and the scripts that use it may expect that certain settings are left on their defaults. If these settings are to be changed, they should be options in the scripts that use scroll-list, not scroll-list itself.

Reply:
I want to add loop scrolling options for chapter-list.lua, but I'm not sure how to change  the script

Reply:
You can add loop scrolling by just setting the list.wrap option. It's already built into scroll-list.

Reply:
Thanks, changed.